docs/4.x/how-piwik-works.md

```diff
@@ -137,60 +163,71 @@ Log data is represented in PHP as `Piwik\Tracker\Visit` objects, and is stored i
 
-    These are stored in the `archive_blob_*` tables. `DataTable` objects are serialized to string and compressed to be stored as `BLOB` in the table.
+    These are stored in the `archive_blob_*` tables. `DataTable` objects are serialized to a string and compressed to be stored as `BLOB` in the table.
 
-    `DataTable` objects stored in the database are named *records* to differentiate them from `DataTable` objects manipulated and returned by Piwik's API that we name *reports*.
+Both numeric and table record objects stored in the database are named *records* to differentiate them from `DataTable` objects manipulated and returned by Matomo's API that we name *reports*.
 
-Every numeric metric or table record is processed and stored at each aggregation level: day, week and month. For example, that means that the "Entry pages" report is processed and stored for every day of the month as well as for every week, month, year and custom date range. Such data is redundant, but that is essential to guarantee fast performances.
+Every numeric metric or table record is processed and stored at each aggregation level: day, week and month. For example, that means that the "Entry pages" report is processed and stored for every day of the month as well as for every week, month, year and custom date range. Such data is redundant, but that is essential to guarantee fast performance when requesting a specific period.
 
-Because Archive data must be fast to query, it is splitted in separate tables per month. We will then have:
+Because Archive data must be fast to query, it is divided in separate tables per month. We will then have:
 
-- `archive_numeric_2014_10`: metrics for October 2014
-- `archive_blob_2014_10`: reports for October 2014
-- `archive_numeric_2014_11`: metrics for November 2014
-- `archive_blob_2014_11`: reports for November 2014
+- `archive_numeric_2021_10`: metrics for October 2021
+- `archive_blob_2021_10`: reports for October 2021
+- `archive_numeric_2021_11`: metrics for November 2021
+- `archive_blob_2021_11`: reports for November 2021
 - …
 
-The contents of the archive tables are explained in detail in the ["Piwik database schema" guide](/guides/database-schema#archive-tables). The archiving process is explained in detail in the ["Archiving" guide](/guides/archiving).
+The contents of the archive tables are explained in detail in the ["Matomo database schema" guide](/guides/database-schema#archive-tables). The archiving process is explained in detail in the ["Archiving" guide](/guides/archiving).
+
+#### Auto archiving versus browser archiving
+
+By default, Matomo generates these reports "on demand" every time they are requested in the browser or through the API. This can slow down Matomo, and therefore it is possible to configure [auto archiving](https://matomo.org/docs/setup-auto-archiving/) (sometimes also referred to as pre-archiving) which will instead generates these reports in the background periodically through a cron.
```

Comment:
```suggestion
By default, Matomo generates these reports "on demand" every time they are requested in the browser or through the API. This can slow down Matomo, and therefore it is possible to configure [auto archiving](https://matomo.org/docs/setup-auto-archiving/) (sometimes also referred to as pre-archiving) which will instead generate these reports in the background periodically through a cron.
```